server/config/nl_page/nl_vars_percapita_ranking.csv

```diff
@@ -4193,74 +4218,26 @@ CumulativeCount_MedicalConditionIncident_COVID_19_ConfirmedCase,,,
 CumulativeCount_MedicalConditionIncident_COVID_19_ConfirmedOrProbableCase,,,
 CumulativeCount_MedicalConditionIncident_COVID_19_PatientDeceased,,,
 CumulativeCount_MedicalTest_ConditionCOVID_19,,,
-CumulativeCount_Vaccine_COVID_19_Administered,,,
+CumulativeCount_Vaccine_COVID_19_Administered,n,,n

```

Comment:
Why `n` here?

Reply:
This SV is listed twice in the file (also at the very end) - which line is correct? @pradh